src/tests/frontend/specs/inner_height.js

```diff
@@ -7,10 +7,11 @@ describe('height regression after ace.js refactoring', function () {
 
   // everything fits inside the viewport
   it('clientHeight should equal scrollHeight with few lines', async function () {
-    const aceOuter = helper.padChrome$('iframe')[0].contentDocument;
-    const clientHeight = aceOuter.documentElement.clientHeight;
-    const scrollHeight = aceOuter.documentElement.scrollHeight;
-    expect(clientHeight).to.be(scrollHeight);
+    await helper.clearPad();
+    const outerHtml = helper.padChrome$('iframe')[0].contentDocument.documentElement;
+    // Give some time for the heights to settle.
+    await new Promise((resolve) => setTimeout(resolve, 100));
```

Comment:
this is to prevent flakiness when clearPad resolves after receiving the ACCEPT_COMMIT, but CSS re-rendering is not finished yet, right? As I would really like to get rid of timeouts, I'd rather use some code that causes the browser to re-render the page, ie use window.getComputedStyle(ace_inner.contentDocument...) right after https://github.com/ether/etherpad-lite/blob/develop/src/tests/frontend/helper/methods.js#L37
```
  await helper.waitForPromise(() => editsNum + 1 === helper.commits.length && window.getComputedStyle(helper.padChrome$('iframe')[0].contentDocument.documentElement.body);
```

However, no big deal, but something to optimize in the future

Reply:
> I'd rather use some code that causes the browser to re-render the page

Today I learned that this is possible. Thanks!